src/WebJobs.Script/Description/FunctionDescriptorProvider.cs

```diff
@@ -336,15 +337,22 @@ protected ParameterDescriptor ParseTimerTrigger(TimerBindingMetadata trigger, Ty
                 triggerParameterType = typeof(string);
             }
 
+            CrontabSchedule.ParseOptions options = new CrontabSchedule.ParseOptions()
+            {
+                IncludingSeconds = true
+            };
+            if (CrontabSchedule.TryParse(trigger.Schedule, options) == null)
+            {
+                throw new ArgumentException(string.Format("'{0}' is not a valid CRON expression.", trigger.Schedule));
```

Comment:
StyleCop seems to have issues with "some" of the C# 6.0 syntax. String interpolation seems to work without problems.